api/v1beta2/foundationdb_status.go

```diff
@@ -134,6 +134,9 @@ type FoundationDBStatusProcessInfo struct {
 	// Excluded indicates whether the process has been excluded.
 	Excluded bool `json:"excluded,omitempty"`
 
+	// Tainted indicates whether the process's knode has been tainted for a while.
```

Comment:
```suggestion
	// Tainted indicates whether the process's knode has been tainted for a while.
```

" for a while.", can we be more specific here? Are we talking about 1 second, 1 hour, 1 day?

Also I don't believe that the FoundationDB cluster status (https://apple.github.io/foundationdb/mr-status.html) has a field `tainted` in the process information. Why do we need this field here?

Reply:
we don't need this.